rocAL/rocAL/source/augmentations/effects_augmentations/node_pixelate.cpp

```diff
@@ -21,29 +21,25 @@ THE SOFTWARE.
 */
 
 #include <vx_ext_rpp.h>
-#include <node_pixelate.h>
-#include <graph.h>
+#include "node_pixelate.h"
 #include "exception.h"
 
 PixelateNode::PixelateNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs) :
         Node(inputs, outputs)
 {
 }
 
```

Comment:
Please move the braces to previous line similar to how it is done for other nodes